src/status_im/ui/screens/multiaccounts/key_storage/views.cljs

```diff
@@ -139,25 +139,28 @@
                                            :height 16}]]])
 
 (defn keycard-upsell-banner []
-  [react/touchable-highlight {:on-press #(.openURL ^js react/linking "https://get-keycard.status.im/")}
-   [react/view {:background-color (if (= :dark @colors/theme) "#2C5955" "#DDF8F4")
-                :border-radius 16
-                :margin 16
-                :padding-horizontal 12
-                :padding-vertical 8
-                :flex-direction :row}
-    [react/view
+  [react/view {:background-color (if (= :dark @colors/theme) "#2C5955" "#DDF8F4")
+               :border-radius 16 :padding-left 12 :padding-right 50 :margin 16}
+   [react/touchable-highlight {:on-press #(.openURL ^js react/linking "https://get-keycard.status.im/")}
+    [react/view {:padding-vertical 8
+                 :flex-direction :row}
      [react/image {:source (resources/get-theme-image :keycard)
                    :resize-mode :contain
                    :style {:width 48
-                           :height 48}}]]
-    [react/view {:flex 1
-                 :margin-left 12}
-     [react/text {:style {:font-size 20
-                          :font-weight "700"}}
-      (i18n/label :t/get-a-keycard)]
-     [react/text {:style {:color (colors/alpha colors/text 0.8)}}
-      (i18n/label :t/keycard-upsell-subtitle)]]]])
+                           :height 48}}]
+     [react/view {:flex 1
+                  :margin-left 12}
+      [react/text {:style {:font-size 20
+                           :font-weight "700"}}
```

Comment:
Why did the linter did not complain about un-aligned maps? Is this allowed?

Reply:
i believe it doesn't complain about un-aligned maps